.travis/install-extension.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+if [ "x${DRIVER_VERSION}" != "x" ]; then
```

Comment:
As a sanity check, I think it would make sense to raise an error at the top of this script if both `$DRIVER_VERSION` and `$DRIVER_BRANCH` are non-empty. As-is, we're likely to compile the driver twice and end up with the branch version (last one wins).

Reply:
I'll change this around to use an `else if` and give precedence to a branch build. This allows us to omit setting `DRIVER_VERSION=""` in the env for branch builds. If none of the drivers can be installed, the following `php --ri` call will fail, but I can also exit with a non-zero code in this script to break the build earlier.